Installer/dolphin-emu-nogui.6

```diff
@@ -0,0 +1,56 @@
+.TH DOLPHIN-EMU-NOGUI 6 "March 4, 2016"
```

Comment:
I don't believe "Installer" is the right place for this (it was meant for the win32 installer NSIS code, and I'm not sure why people starting putting more stuff in there...). Can you move these files to the Data/ directory?